trimmer_decorator.rb

```diff
@@ -0,0 +1,5 @@
+class TrimmerDecorator
+  def correct_name
+    super() unless name.size > 10
```

Comment:
* `Implement a method correct_name that makes sure that the output of @nameable.correct_name has a maximum of 10 characters. If it's longer it should trim the word.`, Kindly implement this requirement the right way ( ie ` `@nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name``)

Reply:
Hello @Davidosky007 

I already wrote the correct_name method inside the parent class and that's why I am using the super() method to call it. I think while using the decorator patterns we should consider the DRY principle that's why I did like his. Kindly, check it again